modules/flowable-engine/src/test/java/org/flowable/engine/test/api/runtime/ProcessInstanceSuspensionTest.java

```diff
@@ -413,172 +343,99 @@ public void testTaskOperationsFailAfterProcessInstanceSuspend() {
         ProcessDefinition processDefinition = repositoryService.createProcessDefinitionQuery().singleResult();
         ProcessInstance processInstance = runtimeService.startProcessInstanceById(processDefinition.getId());
         final org.flowable.task.api.Task task = taskService.createTaskQuery().processInstanceId(processInstance.getId()).singleResult();
-        assertNotNull(task);
+        assertThat(task).isNotNull();
 
         // Suspend the process instance
         runtimeService.suspendProcessInstanceById(processInstance.getId());
 
-        // Yeah, the following is pretty long and boring ... but I didn't have
-        // the patience
+        // Yeah, the following is pretty long and boring ... but I didn't have the patience
         // to create separate tests for each of them.
 
         // Completing the task should fail
-        try {
-            taskService.complete(task.getId());
-            fail("It is not allowed to complete a task of a suspended process instance");
-        } catch (FlowableException e) {
-            // This is good
-        }
+        assertThatThrownBy(() -> taskService.complete(task.getId()))

```

Comment:
On this and the rest of the `assertThatThrownBy` add `as(description)` to keep the description